README.md

```diff
@@ -129,7 +129,7 @@ request for the code review
 - GitHub: [@githubhandle](https://github.com/tsigewossne)
 - Twitter: [@twitterhandle] (https://twitter.com/TsigeWo)
 - Email: [@gmail] (https://tsigedejen@gmail.com)
-- LinkedIn: [LinkedIn](http://ca.linkedin.com/in/linkedintsigewossne)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/feed/?trk=404_page)
```

Comment:
- [ ] This link takes the visitor to his linkedIn profile feed. kindly consider removing this line. Although it is better to mention your linkedIn profile but its okay if you dont have any right now. so consider removing this line and Also in the acknowledgement section, there is a linkedIn link that is broken also, so consider removing it as well.